x-pack/plugins/ingest_manager/scripts/dev_agent/script.ts

```diff
@@ -14,7 +14,11 @@ import {
   PostAgentEnrollRequest,
   PostAgentEnrollResponse,
 } from '../../common/types';
+import * as kibanaPackage from '../../package.json';
 
+// @ts-ignore
```

Comment:
Interesting - what was the TS error?

Reply:
![image](https://user-images.githubusercontent.com/56409205/93109236-cc7a4280-f681-11ea-96c7-38dca0838331.png)